### PR TITLE
data: Parvej2001

### PR DESCRIPTION
**Pull request by Parvej2001**
## Fixes Issue
Created my linkfree portfolio

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7910"></a>

### CODE REVIEW

File: data/Parvej2001.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "bio": "Building Axstudios.tech and Skilled in Full Stack development.Exploring Cloud Native and Blockchain technology with a bachelor of engineering focused in Information Technology",
+  "avatar": "https://github.com/parvej2001.png",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/p_parvez",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Lets collobrate here",
```
Comment (Vishrut19) [inline data/Parvej2001.json:14]: please change the spelling of `collaborate`

File: data/Parvej2001.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      "url": "https://github.com/parvej2001",
+      "icon": "FaGithub"
+    },
+     {
+      "name": "Follow me to hangout and chill",
+      "url": "https://www.instagram.com/parvez_shaikh38",
+      "icon": "FaInstagram"
+    },
```
Comment (Vishrut19) [inline data/Parvej2001.json:22]: Trailing comma. I think there is no need for it

File: data/Parvej2001.json (hunk 1 above)
Comment (Vishrut19) [inline data/Parvej2001.json:14]: I think there is a typo on word `collaborate`

File: data/Parvej2001.json (hunk 2 above)
Comment (Parvej2001) [reply data/Parvej2001.json:22]: Alright 👍 thanks

File: data/Parvej2001.json (hunk 1 above)
Comment (Parvej2001) [reply data/Parvej2001.json:14]: Thank you 😊 for letting me know sir
I will correct it 😄

File: data/Parvej2001.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Parvej Shaikh",
+  "displayStatsPublic" : true,
+  "type":"personal",
```
Comment (SaraJaoude) [inline data/Parvej2001.json:4]: these are no longer needed:
```suggestion
```

File: data/Parvej2001.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  "bio": "Building Axstudios.tech and Skilled in Full Stack development.Exploring Cloud Native and Blockchain technology with a bachelor of engineering focused in Information Technology",
+  "avatar": "https://github.com/parvej2001.png",
```
Comment (SaraJaoude) [inline data/Parvej2001.json:6]: this is no longer needed:
```suggestion
```

Comment (Parvej2001) [reply data/Parvej2001.json:6]: ok thankyou for letting me know. i will change it

Comment (Parvej2001) [reply data/Parvej2001.json]: @SaraJaoude i have done the changes

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**